WooCommerce/Classes/Authentication/Navigation Exceptions/WrongAccountErrorViewModel.swift

```diff
@@ -117,6 +123,32 @@ final class WrongAccountErrorViewModel: ULAccountMismatchViewModel {
                 self?.analytics.track(event: .LoginJetpackConnection.jetpackConnectionErrorShown(selfHostedSite: isSelfHosted))
             }
 
+        $isSelfHostedSite
```

Comment:
Super nit: Would it make sense to move this out of `viewDidLoad` to reduce the number of lines in `viewDidLoad`?